src/ui/public/filter_bar/filter_bar.js

```diff
@@ -46,10 +45,6 @@ module.directive('filterBar', function (Private, Promise, getAppState) {
 
       $scope.state = getAppState();
 
-      // Don't show filter "pinnability" when in embedded mode, as it doesn't make sense in that context
-      // as there will be no cross app navigation for which the filter should persist.
-      $scope.showFilterPin = () => chrome.getVisible();
```

Comment:
Weird.  I had another PR that accidentally got merged into this one, so I reverted it, hence the commit history on this one was a little weird: 1) merge unrelated PR. 2) Do actual work. 3) Notice mistake and revert PR.

But now that original PR is merged for real, and I guess the revert is making this show up as actually removing that code.

Anyway, good catch, I'll add it back in.